cmd/container-disk-v1alpha/BUILD.bazel

```diff
@@ -158,3 +166,17 @@ container_image(
     tars = [":virtio-win-iso-tar"],
     visibility = ["//visibility:public"],
 )
+
+go_library(
+    name = "go_default_library",
+    srcs = ["main.go"],
+    importpath = "kubevirt.io/kubevirt/cmd/container-disk-v1alpha",
+    visibility = ["//visibility:private"],
+    deps = ["//pkg/log:go_default_library"],
+)
+
+go_binary(
+    name = "entry-point.sh",
```

Comment:
eh I know we want to save backward compatibility, but pretend that go binary is a shell script

Reply:
well backward compatibility ...

Anyway, we can wait a little with merging  this. I am almost done with the follow up PR and that does not have this script anymore and is still backward compatible.